### PR TITLE
Common animation code

### LINKED ISSUE #172: Animations: curves & timings
Basic curves and timings we're using:

```javascript
StandardEase = "cubic-bezier(0.4, 0.0, 0.2, 1)"
EaseOut = "cubic-bezier(0.0, 0.0, 0.2, 1)"
EaseIn = "cubic-bezier(0.4, 0.0, 1, 1)"
SharpCurve = "cubic-bezier(0.4, 0.0, 0.6, 1)"
Bounce = "cubic-bezier(0.175, 0.885, 0.32, 1.375)"
```

```javascript
EnterTiming = .225
LeaveTiming = .195
LeaveTimingFaster = .145
delayBetweenEntries = .09
delayBetweenEntriesHide = .04
```

### PR DESCRIPTION
**Pull request by willsb**
Closes #172 

Squash whenever

### CODE REVIEW

File: Toggl.Foundation.MvvmCross/Helper/Animation.cs [hunk 1]
```diff
@@ -0,0 +1,39 @@
+﻿using System;
+
+namespace Toggl.Foundation.MvvmCross.Helper
+{
+    public static class Animation
+    {
+        public struct CubicBezierCurve
```
Comment (amulware) [inline Toggl.Foundation.MvvmCross/Helper/Animation.cs]: What is the point of using a custom type for this?

File: Toggl.Foundation.MvvmCross/Helper/Animation.cs [hunk 2]
```diff
@@ -0,0 +1,39 @@
+﻿using System;
```
Comment (arturs105) [inline Toggl.Foundation.MvvmCross/Helper/Animation.cs]: Using directive is unnecessary

**Comment by willsb** [comment]
Any objections to merging this, @amulware?

**Comment by amulware** [comment]
Go ahead.

**Pull request merged**